content/integrate/write-behind/architecture.md

```diff
@@ -0,0 +1,72 @@
+---
+Title: Write-behind architecture
+aliases: /integrate/redis-data-integration/write-behind/architecture/
+alwaysopen: false
+categories:
+- docs
+- integrate
+- rs
+- rdi
+description: Discover the main components of Write-behind
+group: di
+headerRange: '[2]'
+linkTitle: Architecture
+summary: Redis Data Integration keeps Redis in sync with the primary database in near
+  real time.
+type: integration
+weight: 10
+---
+
+Write-behind lets you integrate Redis Enterprise (as the source of changes to data) and downstream databases or datastores.
+Write-behind captures any changes to a selected set of key patterns in a Redis keyspace and asynchronously writes them in small batches to the downstream database. This means that your app doesn't need to handle the data remodeling or manage the connection with the downstream database.
+
+Write-behind can normalize a key in Redis to several records in one or more tables at the target.
+To learn more about write-behind declarative jobs and normalization, see the
+[write-behind quick start guide]({{< relref "/integrate/write-behind/quickstart/write-behind-guide" >}}).
+
+## Write-behind topology
+
+Write-behind's CLI and engine are shipped as one product that can run both ingest and write-behind pipelines.
+However, the two different types of pipeline have different topologies.
+
+The Write-behind engine is installed on the Redis database that contains the application data and not on a separate staging Redis database. The Write-behind data streams and its control plane add only a small footprint of a few hundred MB to the Redis database. In the write-behind topology, Write-behind processes data in parallel on each shard and establishes a single connection from each shard to the downstream database.
```

Comment:
```suggestion
The Write-behind engine is installed on the Redis database that contains the application data and not on a separate staging Redis database. The Write-behind data streams and its control plane adds only a small footprint of a few hundred MB to the Redis database. In the write-behind topology, Write-behind processes data in parallel on each shard and establishes a single connection from each shard to the downstream database.
```

Reply:
To me, this reads as 2 or more things: the streams and the control plane, so it seems like "they add..."